content/docs/user-guide/gems/reference/ros2/class-diagram.md

```diff
@@ -0,0 +1,13 @@
+---
+linkTitle: Class Diagram
+title: O3DE ROS 2 Gem Details
```

Comment:
`linkTitle` and `title` should be similar, where `linkTitle` is usually a shorter version of `title`. Specifically, `title` acts as a header 1 (`# Title` in Markdown). `linkTitle` is the title that appears on the left-hand navigation.
```suggestion
linkTitle: ROS 2 Class Diagram
title: ROS 2 Class Diagram
```